### PR TITLE
1st two matches from 2021 "base-space tournament"

### PR DESCRIPTION
**Pull request by jordancurve**
Add Burning Skull's openings from his first two matches in the 2021 "Dominion base-space tournament".  Sources:
- Burning Skull, [Dominion base-space tournament vs amoffett11](https://www.youtube.com/watch?v=DorVPvPQQxY), *YouTube* (2021-03-28)
- Burning Skull, [Dominion base-space tournament vs JaCrispyKing](https://www.youtube.com/watch?v=rgmGwpqKpYs), *YouTube* (2021-04-04)

### CODE REVIEW

**Review by zuirod — CHANGES_REQUESTED** [review]
Hey, great idea! Thanks for your contribution. I have a couple of questions:
1. Burning Skull just finished his third and final match. Could you add that?
2. I see two openings for game id 70580237, so one of those should have hypothetical set to true? (hypothetical meaning that he said he would play it, but didn't actually play it in the game)
3. There are 4 games per match, but only 5 new game ids. Which games did you not include?

**Comment by jordancurve** [comment]
Oops, thanks for catching that. I've updated the PR to add the final match, fix the duplicate game ID, and add the missing games. I also added some extra links to the description make it easier to check my work.

File: data/openings.csv [hunk 1]
```diff
@@ -31,8 +31,14 @@ game_id,hypothetical,player,opening_type,opening,kingdom
 6193342,1,2,5-2,Sentry-,Harbinger-Workshop-Bureaucrat-Militia-Poacher-Council Room-Laboratory-Library-Market-Sentry
 6227420,1,2,5-2,Sentry-,Merchant-Gardens-Militia-Moneylender-Remodel-Throne Room-Bandit-Festival-Laboratory-Sentry
 70580237,0,1,3-4,Silver-Militia,Cellar-Moat-Vassal-Village-Militia-Poacher-Festival-Library-Sentry-Witch
-70580237,0,1,2-5,Chapel-Laboratory,Chapel-Moat-Workshop-Bureaucrat-Moneylender-Poacher-Throne Room-Bandit-Laboratory-Witch
+70580796,0,1,2-5,Chapel-Laboratory,Chapel-Moat-Workshop-Bureaucrat-Moneylender-Poacher-Throne Room-Bandit-Laboratory-Witch
```
Comment (zuirod) [inline data/openings.csv]: player should be 2 here

File: data/openings.csv [hunk 2]
```diff
@@ -31,8 +31,14 @@ game_id,hypothetical,player,opening_type,opening,kingdom
...
+70581749,0,1,4-3,Moneylender-Silver,Moat-Vassal-Militia-Moneylender-Poacher-Remodel-Council Room-Festival-Market-Mine
 70582512,0,2,4-3,Chapel-Vassal,Chapel-Harbinger-Vassal-Throne Room-Laboratory-Market-Mine-Sentry-Witch-Artisan
 71112172,0,1,3-4,Vassal-Poacher,Vassal-Village-Bureaucrat-Poacher-Smithy-Throne Room-Council Room-Festival-Library-Sentry
 71113715,0,2,4-3,Poacher-Chapel,Cellar-Chapel-Merchant-Vassal-Village-Poacher-Bandit-Laboratory-Market
```
Comment (zuirod) [inline data/openings.csv]: Kingdom missing Artisan

File: data/openings.csv [hunk 3]
```diff
@@ -31,8 +31,14 @@ game_id,hypothetical,player,opening_type,opening,kingdom
...
+71114568,0,1,4-3,Militia-Chapel,Cellar-Chapel-Moat-Village-Gardens-Militia-Moneylender-Smithy-Laboratory-Sentry
```
Comment (zuirod) [inline data/openings.csv]: Sentry should be Library

File: data/openings.csv [hunk 4]
```diff
@@ -31,8 +31,14 @@ game_id,hypothetical,player,opening_type,opening,kingdom
...
 71115823,0,2,3-4,Silver-Militia,Moat-Merchant-Vassal-Workshop-Gardens-Militia-Library-Market-Sentry-Artisan
+71933307,0,1,5-2,Bandit-Chapel,Cellar-Chapel-Merchant-Vassal-Moneylender-Smithy-Bandit-Council Room-Laboratory-Library
+71934132,0,2,5-2,Workshop-Moat,Chapel-Moat-Merchant-Village-Workshop-Bureaucrat-Gardens-Poacher-Smithy-Library-Mine
```
Comment (zuirod) [inline data/openings.csv]: 11 cards in the kingdom. Village should be removed.

File: data/openings.csv [hunk 5]
```diff
@@ -31,8 +31,14 @@ game_id,hypothetical,player,opening_type,opening,kingdom
...
+71934929,0,1,3-4,Chapel-Militia,Chapel-Harbinger-Militia-Moneylender-Poacher-Remodel-Council Room-Laboratory-Library-Market
+71935274,0,1,3-4,Chapel-Poacher,Chapel-Harbinger-Merchant-Gardens-Poacher-Smithy-Bandit-Laboratory-Market-Witch
```
Comment (zuirod) [inline data/openings.csv]: player should be 2

**Review by zuirod — CHANGES_REQUESTED** [review]
Thanks for adding the final match!

**Comment by jordancurve** [comment]
Done! Thanks for the corrections.

**Review by zuirod — APPROVED** [review]
Great!

**Pull request merged**